### PR TITLE
[HOPSWORKS-716] Unify Spark Configuration for Jobs service and Jupyter

### PR DESCRIPTION
**Pull request by robzor92**
## Make sure there is no duplicate PR for this issue

* **Please check if the PR meets the following requirements**
- [ ] Adds tests for the submitted changes (for bug fixes & features)
- [x] Passes the tests
- [x] HOPSWORKS JIRA issue has been opened for this PR
- [x] All commits have been squashed down to a single commit

* **Post a link to the associated JIRA issue**

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Comment by robzor92** [comment]
https://github.com/logicalclocks/hops-testing/pull/313

**Comment by robzor92** [comment]
@tkakantousis @SirOibaf please review.

File: hopsworks-common/src/main/java/io/hops/hopsworks/common/jobs/spark/DistributionStrategy.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/*
+ * This file is part of Hopsworks
+ * Copyright (C) 2018, Logical Clocks AB. All rights reserved
```
Comment (SirOibaf) [inline hopsworks-common/src/main/java/io/hops/hopsworks/common/jobs/spark/DistributionStrategy.java]: 2019

File: hopsworks-common/src/main/java/io/hops/hopsworks/common/jobs/spark/ExperimentType.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/*
+ * This file is part of Hopsworks
+ * Copyright (C) 2018, Logical Clocks AB. All rights reserved
```
Comment (SirOibaf) [inline hopsworks-common/src/main/java/io/hops/hopsworks/common/jobs/spark/ExperimentType.java]: 2019

File: hopsworks-common/src/main/java/io/hops/hopsworks/common/jobs/spark/DistributionStrategy.java (hunk 1 above)
Comment (robzor92) [reply hopsworks-common/src/main/java/io/hops/hopsworks/common/jobs/spark/DistributionStrategy.java]: Fixed

File: hopsworks-common/src/main/java/io/hops/hopsworks/common/jobs/spark/ExperimentType.java (hunk 2 above)
Comment (robzor92) [reply hopsworks-common/src/main/java/io/hops/hopsworks/common/jobs/spark/ExperimentType.java]: Fixed

**Pull request merged**